packages/react-components/react-swatch-picker-preview/stories/SwatchPicker/SwatchPickerDefault.stories.tsx

```diff
@@ -1,4 +1,33 @@
 import * as React from 'react';
-import { SwatchPicker, SwatchPickerProps } from '@fluentui/react-swatch-picker-preview';
+import { SwatchPicker, ColorSwatch, SwatchPickerSelectEventHandler } from '@fluentui/react-swatch-picker-preview';
 
-export const Default = (props: Partial<SwatchPickerProps>) => <SwatchPicker {...props} />;
+export const Default = () => {
+  const [selectedColor, setSelectedColor] = React.useState('#fff');
+  const handleSelect: SwatchPickerSelectEventHandler = (_, data) => {
+    setSelectedColor(data.selectedValue);
+  };
+  return (
+    <>
+      <SwatchPicker aria-label="SwatchPicker default" selectedValue={selectedColor} onSelectionChange={handleSelect}>
+        <ColorSwatch color="#FF1921" value="#FF1921" aria-label="red" role="radio" />
```

Comment:
+1 , I was confused what's the purpose od these 2 values within spec. Would be good to make it more obvious

Reply:
`value` should be unique. For example, in case of duplicate color if there's no value, two colors will be selected:
![image](https://github.com/microsoft/fluentui/assets/11574680/c8487feb-cf53-4e42-a6c4-53ee3e9c8cf1)
Also, in case with images there might be mapping between small swatches and large images those swatches represent. It's easier to map it by id, not by image URL

Reply:
> small swatches and large images

that makes sense

> For example, in case of duplicate color

wondering what's the use-case ? shouldn't we rather throw error in order to  stop users shipping that kind of UI to production ?

Reply:
I checked that in Office products it's not possible to add duplicate but in Paint it's possible. This options can happen if user sets color using ColorPicker
![image](https://github.com/microsoft/fluentui/assets/11574680/02bffad3-b696-4387-8915-d5ae1d7f5456)